File: FindSubArrayInArray/FindSubArrayInArray.js
function FindSubArrayInArray(arr1, arr2) {
  let hset = new Set();
  if (arr1) {
    for (i = 0; i < arr1.length; i++) {
      if (!hset.has(arr1[i])) {
        hset.add(arr1[i]);
      }
    }
    for (let i = 0; i < arr2.length; i++) {
      if (!hset.has(arr2[i])) {
        return false;
        break;
      }
    }
    return true;
  }
}

if (FindSubArrayInArray([11, 1, 13, 21, 3, 7], [11, 3, 7, 1]))
  console.log("Array 2  is a subset of Array 1 ");
else console.log("Array 2  is not a subset of Array 1");

File: FindSubArrayInArray/FindSubArrayInArray.py
def FindSubArrayInArray(arr1 , arr2):
    hashset = set()
    for i in range(0, len(arr1)):
        hashset.add(arr1[i])
    for i in range(0, len(arr2)):
        if arr2[i] in hashset:
            continue
        else:
            return False
    return True

if(FindSubArrayInArray([ 11, 1, 13, 21, 3, 7 ] , [ 11, 3, 7, 1 ])):
    print("Array 2 is a subset of Array 1")
else:
    print("Array 2 is a not subset of Array 1")
